### PR TITLE
Linked Posts ORM work for Organizers

### PR DESCRIPTION
**Pull request by sc0ttkclark**
https://central.tri.be/issues/116275

### CODE REVIEW

File: src/Tribe/Repositories/Organizer.php [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+		) );
+	}
+
+	/**
+	 * Filter by LIKE or REGEX string.
+	 *
+	 * @param string $meta_key Meta key to filter by.
+	 * @param string $value    MySQL compatible LIKE or REGEX string.
+	 */
+	public function filter_by_like_regex( $meta_key, $value ) {
```
Comment (lucatume) [inline src/Tribe/Repositories/Organizer.php]: This seems a good candidate to be ported back to the base `Tribe__Repository` class.

**Comment by lucatume** [comment]
One refactor and fix tests.

File: src/Tribe/Repositories/Organizer.php (hunk 1 above)
Comment (sc0ttkclark) [reply src/Tribe/Repositories/Organizer.php]: 👍

**Comment by sc0ttkclark** [comment]
Moved into #2222

**Pull request closed without merging**